### ISSUE TITLE
Progressive loading with pagers

### ISSUE THREAD

**Issue by tyssen** [malsup/cycle2#426]
You mention at the bottom of the progressive page that slideshows with pagers are not supported because it seems like an edge case but you might add support for it.

Just wondering if anything had changed since then.

I've used Cycle2 for Galleria-style image galleries and it seems that having the large image loaded only when you click on a thumbnail wouldn't be such an edge case, particularly if the gallery contains a lot of images.

**Comment by malsup** [comment]
Yeah, when I wrote that I was thinking it was an edge case but there is definitely strong demand for that feature.  There is a pull request pending that I need to evaluate but haven't found the time yet.

**Comment by tyssen** [comment]
OK great.

**Comment by jasonday** [comment]
I'm looking at this as well as a need. I'm using the standard pager with progressive loading, and you as the page loads, the pager increases in number of dots. So on init, you just see one pager dot. As the second one loads, then a second dot is added. Very odd UI.

I think you could use the same JSON object for the progressive to create the pager on init, with the correct # and links.

**Comment by tyssen** [comment]
Just wondering what the status of this is.

**Comment by richgcook** [comment]
Me too... any news? @malsup

**Comment by Krindi** [comment]
This particular feature would be of significant value for my company's usage, as the vast majority of my slideshows have pagers.

**Comment by kamov** [comment]
Any news for this feature?
